lints/bin/default_command

```diff
@@ -1,25 +0,0 @@
-#!/usr/bin/env bash
-
-num_errors=0
-
-cd /app/code
-therapist run --use-tracked-files
-if [[ $? -ne 0 ]]; then
-    num_errors=$((num_errors + 1))
-fi
-
-for directory in *; do
-    if [[ -f $directory/package.json ]]; then
-        echo "Checking $directory for known NPM security issues..."
-        pushd $directory
-        # Temporarily disable nsp, as it fails builds if it can't reach the NSP servers
-        # See https://github.com/mozilla/normandy/issues/521
-        nsp check || true
-        if [[ $? -ne 0 ]]; then
-            num_errors=$((num_errors + 1))
-        fi
-        popd
-    fi
-done
```

Comment:
As per offline-discussion, can we re-enable NSP for package.json files that we care about as a lint step?